### ISSUE TITLE
NVDA does not announce bullets in Open Office Writer

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#549]
**Reported by iris on 2010-02-08 14:15**
when navigating through a document in Open Office Writer, NVDA does not announce the presence of bullets on a given page.

**Comment by leonardder** [comment]
I can't reproduce this in LibreOffice 5.3.3.2

**Comment by jcsteh** [comment]
Probably applied to the old Java Access Bridge based accessibility for OpenOffice and LibreOffice, which was replaced by IAccessible2 quite some time ago. Closing as worksforme.
